docs/system_overview/configuration.md

```diff
@@ -86,30 +132,33 @@ This file contains the following top level entries:
 
-  - `server_control_plane_port` (required) (default 1248) - The port on which the control plane operates. This must be the same across all iRODS servers in a Zone.
+  - `server_control_plane_port` (required) (default 1248) - The port on which the control plane operates. This must be the same across all iRODS servers in a Zone
 
   - `server_control_plane_timeout_milliseconds` (required) (default 10000) - The amount of time before a control plane operation will timeout
 
-  - `server_port_range_start` (required) (default 20000) - The beginning of the port range available for parallel transfers. This must be the same across all iRODS servers in a Zone.
-  - `server_port_range_end` (required) (default 20199) - The end of the port range available for parallel transfers.  This must be the same across all iRODS servers in a Zone.
+  - `server_port_range_start` (required) (default 20000) - The beginning of the port range available for parallel transfers. This must be the same across all iRODS servers in a Zone
+
+  - `server_port_range_end` (required) (default 20199) - The end of the port range available for parallel transfers.  This must be the same across all iRODS servers in a Zone
 
-  - `xmsg_port` (default 1279) - The port on which the XMessage Server operates, should it be enabled. This must be the same across all iRODS servers in a Zone.
   - `zone_auth_scheme` (required) - The authentication scheme used by the zone_user: native, PAM, KRB, or GSI
-  - `zone_key` (required) - The shared secret used for authentication and identification of server-to-server communication - this can be a string of any length, excluding the use of hyphens, for historical purposes.  This must be the same across all iRODS servers in a Zone.
-  - `zone_name` (required) - The name of the Zone in which the server participates. This must be the same across all iRODS servers in a Zone.
-  - `zone_port` (required) (default 1247) - The main port used by the Zone for communication. This must be the same across all iRODS servers in a Zone.
-  - `zone_user` (required) - The name of the rodsadmin user running this iRODS instance.
 
+  - `zone_key` (required) - The shared secret used for authentication and identification of server-to-server communication - this can be a string of any length, excluding the use of hyphens, for historical purposes.  This must be the same across all iRODS servers in a Zone
+
+  - `zone_name` (required) - The name of the Zone in which the server participates. This must be the same across all iRODS servers in a Zone
+
+  - `zone_port` (required) (default 1247) - The main port used by the Zone for communication. This must be the same across all iRODS servers in a Zone
+
+  - `zone_user` (required) - The name of the rodsadmin user running this iRODS instance
 
 Servers in the catalog provider role have the following additional top level entries related to the database connection:
 
```

Comment:
these next ones are not top-level, they live in the database plugin section...

Reply:
done